packages/iov-ethereum/src/erc20.ts

```diff
@@ -11,12 +11,12 @@ export interface EthereumRpcClient {
 
 export interface Erc20Options {
   readonly contractAddress: Address;
-  /** Override on-chain symbol. Use this of contract does not define value on-chain */
-  readonly symbol?: string;
-  /** Override on-chain name. Use this of contract does not define value on-chain */
+  /** The token ticker. Overrides the on-chain value. */
+  readonly symbol: string;
+  /** The number of fractional digits. Overrides the on-chain value. */
+  readonly decimals: number;
```

Comment:
Why is `name` optional but `decimals` and `symbol` not?

Reply:
Because we need `decimals` and `symbol` in the decoder (for `Amount`).